apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex

```diff
@@ -171,6 +172,21 @@ defmodule EthereumJSONRPC.Transaction do
     elixir_to_params(%{transaction | "input" => "0x"})
   end
 
+  # eip-1559 typed transaction parsing
+  def elixir_to_params(%{"type" => "0x0"} = transaction),
+    do: TransactionParsing.parse_legacy_transaction(transaction)
+
+  def elixir_to_params(%{"type" => "0x1"} = transaction),
+    do: TransactionParsing.parse_access_list_transaction(transaction)
+
+  def elixir_to_params(%{"type" => "0x2"} = transaction),
+    do: TransactionParsing.parse_dynamic_fee_transaction(transaction)
+
+  def elixir_to_params(%{"type" => "0x7c"} = transaction),
+    do: TransactionParsing.parse_celo_transaction(transaction)
+
+  # fall back to legacy parsing when type does not match
+
   def elixir_to_params(
```

Comment:
Perhaps, you can at least remove this clause and the next one because of the `type` matching. That's a [recent change](https://github.com/celo-org/blockscout/commit/5900c7cc4dc8ee4ac7350f7e468a140241e23a74#diff-5d997faca6cb09a38fe5ed9c60d34474067710064d51d631709014adaa68a060) from upstream that doesn't make sense to keep if we have an alternative typed parsing implemented.